docs/services/azure-services-guide.md

```diff
@@ -0,0 +1,530 @@
+    namespace: azure
+  reclaimPolicy: Delete
+```
+
+* Create the `SQLServerClass`:
+```bash
+sed "s/RESOURCE_GROUP_NAME/$RESOURCE_GROUP_NAME/g" azure-mysql-class.yaml | kubectl apply -f -
+```
+
+* You should see the following output:
+```bash
+sqlserverclass.database.azure.crossplane.io/standard-azure-mysql created
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get sqlserverclasses -n azure
+```
+*Output*
+```bash
+NAME                   PROVIDER-REF   RECLAIM-POLICY   AGE
+standard-azure-mysql   demo-azure     Delete           11s
+```
+
+You are free to create more Azure `SQLServerClass` instances to define more potential configurations. For instance, you may create `large-azure-mysql` with field `storageGB: 100`.
+
+#### Namespaces
+Kubernetes namespaces allow for separation of environments within your cluster. You may choose to use namespaces to group resources by team, application, or any other logical distinction. For this demo, we will create a namespace called `app-project1-prod`, which we will use to group our Wordpress resources.
+
+* Define a `Namespace` in `app-project1-prod-namespace.yaml`:
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: app-project1-prod
+```
+
+* Create the `Namespace`:
+```bash
+kubectl create -f app-project1-prod-namespace.yaml
+```
+
+* You should see the following output:
+```bash
+namespace/app-project1-prod created
+```
+
+#### Portable Resource Classes
+Portable resource classes are used to define a class of service in a single namespace for an abstract service type. We want to define our Azure `SQLServerClass` as the standard MySQL class of service in the namespace that our Wordpress resources will live in.
+
+* Define a `MySQLInstanceClass` in `mysql-class.yaml` for namespace `app-project1-prod`:
+```yaml
+---
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstanceClass
+metadata:
+  name: standard-mysql
+  namespace: app-project1-prod
+classRef:
+  kind: SQLServerClass
+  apiVersion: database.azure.crossplane.io/v1alpha2
+  name: standard-azure-mysql
+  namespace: azure
+```
+
+* Create the `MySQLInstanceClass`:
+```bash
+kubectl apply -f mysql-class.yaml
+```
+
+* You should see the following output:
+```bash
+mysqlinstanceclass.database.crossplane.io/standard-mysql created
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get mysqlinstanceclasses -n app-project1-prod
+```
+*Output*
+```bash
+NAME             AGE
+standard-mysql   27s
+```
+
+Once again, you are free to create more `MySQLInstanceClass` instances in this namespace to define more classes of service. For instance, if you created `large-azure-mysql` above, you may want to create a `MySQLInstanceClass` named `large-mysql` that references. You may also choose to create MySQL resource classes for other non-Azure providers, and reference them for a class of service in the `app-project1-prod` namespace.
```

Comment:
large-azure-mysql -> azure-mysql-large (cloud-specific class)

large-mysql -> mysql-large (portable class)

app-project1-prod -> app-project1-dev

for consistency with https://github.com/crossplaneio/crossplane/blob/7fbcc6acdf3bd7c4bd1de4a45864e445e074dc14/docs/using-services.md#overview-of-the-cloud-specific-guides